x-pack/plugins/apm/public/selectors/throuput_chart_selectors.ts

```diff
@@ -45,15 +44,6 @@ export function getThroughputTimeseries({
   const bucketKeys = throughputTimeseries.map(({ key }) => key);
   const getColor = getColorByKey(bucketKeys, theme);
 
-  if (!throughputTimeseries.length) {
-    const start = throughputTimeseries[0].dataPoints[0].x;
-    const end =
-      throughputTimeseries[0].dataPoints[
-        throughputTimeseries[0].dataPoints.length - 1
-      ].x;
-    return getEmptySeries(start, end);
```

Comment:
why did we have this?

Reply:
Before my change to split the charts API into latency and throughput APIs, when the `tpmBuckets` were empty it used the `avg` bucket to create an empty series, but it is not needed at all, Elastic charts will show the empty message when the series is equals to `[]`. So we can just skip this and return the timeseries with empty data.

https://github.com/elastic/kibana/blob/b46655d7f6c387755cde0bd93c02201efa470190/x-pack/plugins/apm/public/selectors/chart_selectors.ts#L171-L177